src/main/java/net/imagej/ops/stats/StatsNamespace.java

```diff
@@ -90,6 +93,44 @@
 		return result;
 	}
 
+	@SuppressWarnings("rawtypes")
+	@OpMethod(op = net.imagej.ops.stats.IntegralMean.class)
+	public DoubleType integralMean(final DoubleType out,
+		final RectangleNeighborhood in1, final Interval in2)
+	{
+		final DoubleType result = (DoubleType) ops().run(
+			net.imagej.ops.stats.IntegralMean.class, out, in1, in2);
+		return result;
+	}
+
+	@SuppressWarnings("rawtypes")
+	@OpMethod(op = net.imagej.ops.stats.IntegralSum.class)
+	public DoubleType integralSum(final RectangleNeighborhood in) {
+		final DoubleType result = (DoubleType) ops().run(
+			net.imagej.ops.stats.IntegralSum.class, in);
+		return result;
+	}
+
+	@SuppressWarnings("rawtypes")
+	@OpMethod(op = net.imagej.ops.stats.IntegralSum.class)
+	public DoubleType integralSum(final DoubleType out,
```

Comment:
You are right. However, the requirements for `integral*` methods are deviating from a typical `sum` method so much that I don't think the framework would have any chance to bring those together.

That is mostly due to the fact that e.g. `integralSum` cannot be fed a typical `II` to compute the sum but rather requires computation of an integral image beforehand. We could make `integralSum` behave in a more `sum`-like way by internally computing the integral image from the input `RAI`. But then you (as developer) would not be able to reuse the integral image, which is the whole point of integral image-backed implementations..